commands/instances.go

```diff
@@ -676,3 +677,28 @@ func createInstance(ctx context.Context, getLibOnly bool) (*createInstanceResult
 
 	return res, nil
 }
+
+// SketchLoad collects and returns all files composing a sketch
+func SketchLoad(ctx context.Context, req *rpc.SketchLoadReq) (*rpc.SketchLoadResp, error) {
+	sketch, err := builder.SketchLoad(req.SketchPath, "")
+	if err != nil {
+		return nil, fmt.Errorf("Error loading sketch: %v", err)
```

Comment:
Can we make `req.SketchPath` as part of the error message?

Reply:
Yes, we could but think that some informationg is already in `err`.